server/views/admin-list-articles.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <div class="row justify-content-md-center">
+
+     
+
+
```

Comment:
We can format this document a bit better and remove all this necessary whitespace (both above and below)